README.md

```diff
@@ -81,6 +81,17 @@ To run Google Lighthouse in headless mode and generate a JUnit report:
 A11Y_CONFIG=iplayer-web/all npm run start:lighthouse:junit-headless
 ```
 
+### Run Google Lighthouse with custom configs, e.g. simorgh/simorgh
+
+Lighthouse can be configured to run Progressive Web App, SEO, and Best Practice tests. See the config file in `config/simorgh/simorgh.js`. `a11y-tests-web` will generate a summary report in JSON, saved to `a11y-tests-web/lighthouse-report.json`. An OUTPUT_JSON path can also be specified.
+
+Run lighthouse tests and save to OUTPUT_JSON path
+```
+A11Y_CONFIG=simorgh/simorgh OUTPUT_JSON='/../lighthouse-report.json' npm run start:lighthouse:junit 
```

Comment:
I'd prefer `OUTPUT_JSON` to be called `A11Y_` something, so that all the environment variables begin `A11Y` to avoid potential conflict.

Reply:
Is the JSON report in addition to or instead of the JUnit? The command suggests it's just `junit` but then the environment variable suggests it's JSON. What's the use case for JSON? What's the use case for both?

Reply:
Re `OUTPUT_JSON` is prefixed with OUTPUT to indicate the where the `lighthouse-report.json` should be written and its filename. The reason I chose to begin with OUTPUT is because the thing that is output is the result of testing one or more Lighthouse categories, and it is not limited to a11y tests. Let me know if that would be OK.

The use case for JSON is that is a succinct summary of results per Lighthouse category. Also, in a second PR (branch `parsing`), that JSON is consumed, along with threshold configs that are passed in to a config, and the process exists with code 1 if the threshold for a given category (e.g. a11y 100%) isn't met. This would allow our Travis builds to fail code that doesn't meet our standards. I think the use case for `junit` output is that it is used by Jenkins. We are also using Jenkins.

Reply:
Wrt `OUTPUT_JSON`, I can understand why you've not prefixed it with `A11Y` given it's not limited to `A11Y`. My only concern is this may clash with other environment variables a user may have for another application. I think whilst this repo is still called `a11y-tests-web` (and therefore includes a11y in the name), prefixing this variable with `A11Y` is probably fine, unless you can think of a better prefix for all the environment variables (maybe `ATW`?).

Wrt to the JSON output, if it's not useful at this stage (i.e. before the other PR is done), I suggest pulling it out until it's useful, unless I've misunderstood what you're using it for?

Reply:
Hi @andymsuk I understand your reservation to not use `OUTPUT_JSON`. `ATW` sounds like a sensible prefix for environment variables but I am aware that it would be a breaking change that would require a major version bump. Is that something we want to do right now?

Reply:
Happy to do a major version bump :)